core/Cargo.toml

```diff
@@ -20,7 +20,7 @@ default = ["std"]
 std = ["math/std", "winter-utils/std", "winter-crypto/std"]
 
 [dependencies]
-math = { package = "winter-math", version = "0.4.2", default-features = false }
-crypto = { package = "miden-crypto", version = "0.1.2", default-features = false }
-winter-crypto = { package = "winter-crypto", version = "0.4.2", default-features = false }
-winter-utils = { package = "winter-utils", version = "0.4.2", default-features = false }
+math = { package = "winter-math", version = "0.5.1", default-features = false }
+crypto = { package = "miden-crypto", version = "0.1.3", default-features = false }
+winter-crypto = { package = "winter-crypto", version = "0.5.1", default-features = false }
+winter-utils = { package = "winter-utils", version = "0.5.1", default-features = false }
```

Comment:
Can we get rid of `winter-crypto` import here and get everything from `miden-crypto`?

Reply:
Ditto; deps graph simplified